### DISCUSSION TITLE
Version 4.0

### DISCUSSION THREAD

**Discussion by tanmay2022** [ICME-India/MicroSim#45] [General]
This release contains the updates till the 4th workshop during January, 2024.

<hr /><em>This discussion was created from the release <a href='https://github.com/ICME-India/MicroSim/releases/tag/v4.0'>Version 4.0</a>.</em>

**Comment by drNoahMahammad** [comment]
Hello, do you need a Makefile, and converting this to an Openacc executable? 
![image](https://github.com/ICME-India/MicroSim/assets/171016369/f251651e-4a73-4098-a6c6-d2a7dc0af9db)
